docs/proposals/35-observability2.0-victoria-logs.md

```diff
@@ -0,0 +1,231 @@
+This pattern will remain and log shippers will only be expected to use the `OTLP` protocol, instead of the `Loki` protocol, via HTTP when inserting logs to VictoriaLogs.
+In the `Garden` cluster, the only log shippers are the `FluentBit` instances. They will be configured to send logs via `OTLP`.
+
+In other words, the `VictoriaLogs` instance will be exposed via a service in the `garden` namespace.
+
+### 2. Deployment of VictoriaLogs on Seed Clusters
+
+Analogously to the deployment in the `Garden` cluster, this includes 2 steps:
+- Deployment of a new [VictoriaOperator](https://github.com/VictoriaMetrics/operator) component during the `Seed` reconciliation flow in the `garden` namespace.
+- Creation of a `VLSingle` CustomResource in the `garden` namespace during the `Seed` reconciliation flow but *after* the `VictoriaOperator` k8s deployment has finished.
+The `VLSingle` CustomResource will be managed by the operator and will create a `VictoriaLogs` k8s deployment in the `garden` namespace.
+
+#### Sending Logs to VictoriaLogs in the `Seed` cluster
+
+Since the setup is the same as in the `Garden` cluster, see details in [Sending Logs to VictoriaLogs in the `Garden` cluster](#sending-logs-to-victorialogs-in-the-garden-cluster).
+
+### 3. Deployment of VictoriaLogs in the Shoot Control Plane
+
+During the `Shoot` reconciliation flow, a `VLSingle` CustomResource will be created in the control-plane namespace of the `Shoot` cluster.
+The operator will manage the `VLSingle` CustomResource and create a `VictoriaLogs` k8s deployment.
+
+Example `VLSingle` CR:
+```
+apiVersion: operator.victoriametrics.com/v1
+kind: VLSingle
+metadata:
+  name: example
+  namespace: shoot--local--local
+spec:
+  storage:
+    resources:
+      requests:
+        storage: 30Gi
+  resources:
+    requests:
+      memory: 100Mi
+      cpu: 10m
+    limits:
+      memory: 300Mi
+      cpu: 50m
+```
+
+#### Sending Logs to VictoriaLogs in the `Shoot` control plane
+
+Currently, `Vali` access is behind a service in the namespace of the `Shoot` control plane.
+For external access, there exists an ingress in front of the `OpenTelemetry Collector` in the control-plane of `Shoot` clusters that is used to post logs to `Vali`.
+See [GEP-34](./34-observability2.0-opentelemtry-operator-and-collectors.md) for more details.
+
+The same setup will continue with `VictoriaLogs`, with the only difference being that the `OpenTelemetry Collector` will be configured to ship logs to `VictoriaLogs` via `OTLP` instead of via the `Loki` protocol.
+
+### 4. Migration from Vali to VictoriaLogs
+
+#### Data Migration
+During all the replacements of `Vali` we want to ensure that existing clusters do not get disrupted.
+For this reason, 2 strategies for migration have been considered:
+- "One-shot" migration.
+Reformat all the Vali data into the VictoriaLogs format (or OTLP).
+Could be done via the OpenTelemetry Collector since there is no tool that exists to do this out of the box.
```

Comment:
This sounds a bit contradictory. If it can be done via the collector why would I need a tool? I do not really understand this sentence and the corresponding cons point.